KMB/make_KMB_info.py

```diff
@@ -494,11 +606,25 @@ def make_tag_categories(self, cache):
         tag_map = self.kmb_info.mappings['tags']
         country_map = self.kmb_info.mappings['countries']
         for tag in self.tag:
+            if (self.fmis and tag == 'Fornminnen') or \
+                    (self.bbr and tag.startswith('Byggnadsminnen')):
+                # avoid similar cat to fallback in make_commonscat_categories
+                continue
+
             if tag in tag_map:
                 cat = None
                 if not self.land or self.land == 'se' and \
                         tag_map[tag].get('SE'):
                     cat = tag_map[tag].get('SE')
+
+                    # attempt municipal categorisation
+                    if self.kommunName:
+                        test_cat = tag_map[tag].get('SE').replace(
+                            'Sweden', '{} Municipality'.format(
+                                self.kommunName))
+                        if self.kmb_info.category_exists(test_cat, cache):
+                            self.needs_place_cat = False
```

Comment:
On [L381](https://github.com/lokal-profil/upload_batches/pull/11/files/39a31a810c924e0bc3c0f8c35f2d4f6883d97274#diff-c27a0e60250eb4480e9cb3a67a168625R381) the plan is to introduce a mechanism which adds the municipal or parish category (if Sweden) and possibly a country category otherwise.